### PR TITLE
Jcc/modify 1

### PR DESCRIPTION
**Pull request by jona125**
Add translation.jl and test/register_test.jl
Clear files in src, but not yet with test cases.

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -1,29 +1,20 @@
-name = "REALMSupport"
-uuid = "926d8977-77e3-4bba-bc27-2e234c0c0038"
-authors = ["Jonathan Chang <jona125@gmail.com> and contributors"]
-version = "0.1.0"
```
Comment (timholy) [inline Project.toml:4]: What's the reason for deleting these?

File: Project.toml [hunk 2]
```diff
@@ -1,29 +1,20 @@
...
 Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
-
-[compat]
-julia = "1"
-
-[extras]
-Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
-
-[targets]
-test = ["Test"]
```
Comment (timholy) [inline Project.toml:29]: I can help get these updated, I have a PR ready but perhaps we should get this PR merged first.

**Review by timholy — COMMENTED** [review]
Just a couple of comments

File: Project.toml (hunk 1 above)
Comment (jona125) [reply Project.toml:4]: I think this is automatic generated. Meaning some portion of the dependency package management is incorrect?

Comment (timholy) [reply Project.toml:4]: I can't imagine what would do that (and haven't seen it happen before), but obviously not intentional. If you notice it again, let me know if you figure out what triggers it.

**Pull request merged**